content/en/get-started.md

```diff
@@ -9,20 +9,24 @@ The easisest way to don't miss a thing on the latest trends in OSPOs, job positi
 
 TODO uses slack for real-time chat. In there you will find our #general channel and many regional-focused and working groups focused channels within it ([check TODO group's community structure to learn more](https://github.com/todogroup/governance/blob/main/TODO%20Structure/structure-mindmap.md)). We also have a [OSPO forum via GitHub Discussions](https://github.com/todogroup/ospology/discussions) where you can explore some of the conversations going on in the community.
 
+{{< spacer >}}
```

Comment:
I don't love the use of `spacer` here. I guess it's ok as a one-off, however, I think the theme styles should not require spacers to lay out standard content like we see on this page. I don't think using spacers is common in Hugo-land (like it is in WordPress land).

Reply:
The buttons have margin attached to them so without the spacing I felt the buttons made the page spacing awkward. If the button were not in the middle of the content flow then the spacers would not be needed.